client/src/pages/Profile/Profile.jsx

```diff
@@ -509,49 +538,136 @@ const ProfilePageLeaderPermissionDashboardLinks = () => {
+          >
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              viewBox="0 0 512 512"
+              className={'profile-leader-dashboard-data-icon'}
+            >
+              <path d="M64 256V160H224v96H64zm0 64H224v96H64V320zm224 96V320H448v96H288zM448 256H288V160H448v96zM64 32C28.7 32 0 60.7 0 96V416c0 35.3 28.7 64 64 64H448c35.3 0 64-28.7 64-64V96c0-35.3-28.7-64-64-64H64z" />
+            </svg>
+            <h3 className={'profile-leader-dashboard-data-text'}>Data</h3>
+            <div className={`dashboard-dropdown-image${openDataDropdown ? ' open' : ''}`}>
+              {darkMode ? (
+                <img alt={'arrow'} src={ArrowDarkMode} className="dashboard-dropdown-arrow" />
+              ) : (
+                <img alt={'arrow'} src={Arrow} className="dashboard-dropdown-arrow" />
+              )}
+            </div>
+          </div>
+          <DashboardDropdown
+            open={openDataDropdown}
+            setOpen={setOpenDataDropdown}
+            items={dashboardItems.data}
+            title={'data'}
+          />
+        </div>
+
+        <div className={'profile-leader-dashboard-outreach-links'}>
+          <div
+            className={'profile-leader-dashboard-outreach-title'}
+            onClick={() => setOpenOutreachDropdown(!openOutreachDropdown)}
+          >
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              viewBox="0 0 640 512"
+              className={'profile-leader-dashboard-outreach-icon'}
+            >
+              <path d="M544 248v3.3l69.7-69.7c21.9-21.9 21.9-57.3 0-79.2L535.6 24.4c-21.9-21.9-57.3-21.9-79.2 0L416.3 64.5c-2.7-.3-5.5-.5-8.3-.5H296c-37.1 0-67.6 28-71.6 64H224V248c0 22.1 17.9 40 40 40s40-17.9 40-40V176c0 0 0-.1 0-.1V160l16 0 136 0c0 0 0 0 .1 0H464c44.2 0 80 35.8 80 80v8zM336 192v56c0 39.8-32.2 72-72 72s-72-32.2-72-72V129.4c-35.9 6.2-65.8 32.3-76 68.2L99.5 255.2 26.3 328.4c-21.9 21.9-21.9 57.3 0 79.2l78.1 78.1c21.9 21.9 57.3 21.9 79.2 0l37.7-37.7c.9 0 1.8 .1 2.7 .1H384c26.5 0 48-21.5 48-48c0-5.6-1-11-2.7-16H432c26.5 0 48-21.5 48-48c0-12.8-5-24.4-13.2-33c25.7-5 45.1-27.6 45.2-54.8v-.4c-.1-30.8-25.1-55.8-56-55.8c0 0 0 0 0 0l-120 0z" />{' '}
+            </svg>
+            <h3 className={'profile-leader-dashboard-outreach-text'}>Outreach</h3>
+            <div className={`dashboard-dropdown-image${openOutreachDropdown ? ' open' : ''}`}>
+              {darkMode ? (
+                <img alt={'arrow'} src={ArrowDarkMode} className="dashboard-dropdown-arrow" />
+              ) : (
+                <img alt={'arrow'} src={Arrow} className="dashboard-dropdown-arrow" />
+              )}
+            </div>
+          </div>
+          <DashboardDropdown
+            open={openOutreachDropdown}
+            setOpen={setOpenOutreachDropdown}
+            items={dashboardItems.outreach}
+            title={'outreach'}
+          />
+        </div>
+
+        <div className={'profile-leader-dashboard-scunt-links'}>
```

Comment:
since all three dropdown menus use almost the same code, you can use map and iterate through a list of objects! it would also be best to put the drop downs in a separate folder under the components folder (just to reduce the amount of code in Profile.jsx)

here's one format for your object...similar to the sponsors page, `client\src\util\sponsors.jsx`. 
```jsx
{
  label: "data", // data as an example, it would probably be data, outreach & scunt, this would be used as your key when mapping and also when you use your classnames
  title: "Data", // this can be the title of the dropdown menu
  state: // add the state here
  setState: // add state handler here (or the function that you use in your onClick) 
  icon: // import svg instead of using the tag
}
```

note that since you're using event handlers, you'll have to declare the list of objects within the actual component similar to what you've done for `dashboardItems`.

for importing svgs, you can use them similar to how you use images, using the `<img>` tag, and using `src`, using the `<svg>` tag tends to look a little messy.

you can use this as a reference as well for all the auth scopes, `client\src\pages\ScopeRequest\functions.jsx`